examples/src/main/java/org/neo4j/docs/driver/RxTransactionFunctionExample.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.docs.driver;
+
+import io.reactivex.Flowable;
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.neo4j.driver.reactive.RxResult;
+import org.neo4j.driver.reactive.RxSession;
+import org.neo4j.driver.summary.ResultSummary;
+
+public class RxTransactionFunctionExample extends BaseApplication
+{
+    public RxTransactionFunctionExample( String uri, String user, String password )
+    {
+        super( uri, user, password );
+    }
+
+    public Publisher<ResultSummary> printAllProductsReactor()
```

Comment:
Maybe return the actual implementation here? In the end it's what we wanna demonstrate with the example. 
Would also fit better with the tests for the example.